drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSelectedColumnCommand.java

```diff
@@ -103,8 +112,35 @@ protected void setInstanceHeader(ScenarioSimulationContext context, ScenarioGrid
         final FactIdentifier factIdentifier = setEditableHeadersAndGetFactIdentifier(context, selectedColumn, alias, fullClassName);
         setInstanceHeaderMetaData(selectedColumn, alias, factIdentifier);
         final ScenarioHeaderMetaData propertyHeaderMetaData = selectedColumn.getPropertyHeaderMetaData();
-        setPropertyMetaData(propertyHeaderMetaData, getPropertyPlaceHolder(columnIndex), false, selectedColumn, ScenarioSimulationEditorConstants.INSTANCE.defineValidType());
-        context.getModel().updateColumnInstance(columnIndex, selectedColumn);
+        if (Objects.equals(FactMappingValueType.RAW, factMappingValueType)) {
```

Comment:
@yesamer 
by-side note: Objects.equals is needed when **both** terms of comparison may be null; but since an enum can't be null (FactMappingValueType.RAW), the above could be simply written with `FactMappingValueType.RAW.equals(factMappingValueType)` (and the factMappingValueType possibly `null`). 
Mind you the other way around is wrong, though.

Reply:
@yesamer Thank you for the lesson, I totally forgot I'm a Java developer for a while.... . 
Jokes aside, I perfectly know the scope of it, usually I prefer to use it because I think it's more readable and more "functional".

Reply:
:+1: